src/plugins/dashboard/public/application/dashboard_state_manager.ts

```diff
@@ -103,22 +99,26 @@ export class DashboardStateManager {
     kibanaVersion,
     kbnUrlStateStorage,
     history,
+    usageCollection,
   }: {
     savedDashboard: SavedObjectDashboard;
     hideWriteControls: boolean;
     kibanaVersion: string;
     kbnUrlStateStorage: IKbnUrlStateStorage;
     history: History;
+    usageCollection?: UsageCollectionSetup;
   }) {
     this.history = history;
     this.kibanaVersion = kibanaVersion;
     this.savedDashboard = savedDashboard;
     this.hideWriteControls = hideWriteControls;
+    this.usageCollection = usageCollection;
 
     // get state defaults from saved dashboard, make sure it is migrated
     this.stateDefaults = migrateAppState(
-      getAppStateDefaults(this.savedDashboard, this.hideWriteControls),
-      kibanaVersion
+      { ...getAppStateDefaults(this.savedDashboard, this.hideWriteControls) },
```

Comment:
Wonder what has changed that before this spread wasn't needed? 
Or is this fixes some existing bug?

Reply:
Not sure what caused this, but TypeScript started hate the previous one

Reply:
I tried to revert and seems like all is good. maybe you had some type issues in a process of moving staff around. Could you double check please?

Reply:
Okay, that was my fault.
VSCode accidentally thought that I want to use the TypeScript version which it likes more 🙂 

![typescript_version](https://user-images.githubusercontent.com/31325372/78383477-1f36d500-75e1-11ea-83c8-13685d0d8e01.png)

And throws me such an issue:

![typescript_issue](https://user-images.githubusercontent.com/31325372/78383522-3675c280-75e1-11ea-9ab1-406dbb549828.png)

I reverted back those changes, but seems this will become actual again once ts 3.8.2 is merged

https://github.com/elastic/kibana/pull/57774/files#diff-2f67bccb7b0ca326ff2a7aec1b0ee965